config.js

```diff
@@ -119,6 +120,13 @@ Config.prototype._seed = function _seed(seed) {
     }, 'expected to be array of RegExp objects');
     seedOrDefault('membershipUpdateRollupEnabled', false);
     seedOrDefault('tchannelRetryLimit', 0);
+    // How long a request is outstanding until is it considered
+    // wedged and then canceled.
+    seedOrDefault('wedgedRequestTimeout', 30 * 1000);
```

Comment:
does it make sense to reuse the constant from [`client.js`](https://github.com/uber/ringpop-node/pull/238/files#diff-215b25d5bf5c0b4623ad1a2b62456f60R36). Now we have `30s` defined twice.